tv_apps/tv-epg/js/pin_card.js

```diff
@@ -0,0 +1,44 @@
+/* global evt, CardManager */
```

Comment:
Please move this file to shared and use it. BTW, it would be nice to check if app deck also need this file. If yes, we should leverage this module.